src/odemis/model/_core.py

```diff
@@ -219,24 +218,35 @@ def load_roattributes(self, roattributes):
     # save the list in case we need to pickle the object again
     self._odemis_roattributes = list(roattributes.keys())
 
-def prepare_to_listen_to_more_vas(inc):
-    """
-    There's a limit on the number of VA subscribers we can create (the number of open
-    file descriptors). By default, it's set to 1024 on ubuntu. If we want to use more
-    subscribers, we need to explicitly increase this limit. This is for example necessary in
-    the SettingsObserver (cf odemis.acq).
-    This function allows us to use an additional amount of inc VA subscribers. It corresponds
-    to the system call ulimit -n.
-    inc (int): how many files to open additionally
-    """
-    cur_limit = resource.getrlimit(resource.RLIMIT_NOFILE)
-    try:
-        resource.setrlimit(resource.RLIMIT_NOFILE, (cur_limit[0] + inc, cur_limit[1]))
-    except ValueError:
-        # this happens when starting odemis from eclipse
-        logging.warning("Maximum number of open files is already at its limit %s." % cur_limit[0])
 
-# Container management functions and class
+if os.name != 'nt':
+    import resource
+
+    def prepare_to_listen_to_more_vas(inc):
+        """
+        There's a limit on the number of VA subscribers we can create (the number of open
+        file descriptors). By default, it's set to 1024 on ubuntu. If we want to use more
+        subscribers, we need to explicitly increase this limit. This is for example necessary in
+        the SettingsObserver (cf odemis.acq).
+        This function allows us to use an additional amount of inc VA subscribers. It corresponds
+        to the system call ulimit -n.
+        inc (int): how many files to open additionally
+        """
+
+        cur_limit = resource.getrlimit(resource.RLIMIT_NOFILE)
+        try:
+            resource.setrlimit(resource.RLIMIT_NOFILE, (cur_limit[0] + inc, cur_limit[1]))
+        except ValueError:
+            # this happens when starting odemis from eclipse
+            logging.warning("Maximum number of open files is already at its limit %s." % cur_limit[0])
+
+    # Container management functions and class
```

Comment:
Please leave this comment out of the `if`. It should be just before the next `class`.